lib/cylc/task_proxy.py

```diff
@@ -18,6 +18,7 @@
 
 """Provide a class to represent a task proxy in a running suite."""
 
+from collections import Counter
```

Comment:
`collections.Counter` is a Python2.7 feature!

https://docs.python.org/2/library/collections.html